ImageData/Hdf5Loader.h

```diff
@@ -399,6 +399,36 @@ bool Hdf5Loader::GetRegionSpectralData(int stokes, int region_id, const casacore
         auto t_start = std::chrono::high_resolution_clock::now();
         auto t_latest = t_start;
 
+        // Lambda to calculate mean, sigma and RMS
+        auto calculate_means = [&]() {
```

Comment:
not sure `calculate_means` is an accurate label, since it's doing mean, rms and sigma. Perhaps `calculate_stats` or something else indicating that we're doing more than calculating the mean?

Reply:
Also done.